lib/api/model/events.dart

```diff
@@ -265,6 +267,89 @@ class MessageEvent extends Event {
   }
 }
 
+/// A Zulip event of type `update_message`.
+@JsonSerializable(fieldRename: FieldRename.snake)
+class UpdateMessageEvent extends Event {
+  @override
+  String get type => 'update_message';
+
+  final int? userId; // TODO(server-5)
+  final bool? renderingOnly; // TODO(server-5)
+  final int messageId;
+  final List<int> messageIds;
+  final List<String> flags;
+  final int? editTimestamp; // TODO(server-5)
+  final String? streamName;
+  final int? streamId;
+  final int? newStreamId;
+  final String? propagateMode;
+  final String? origSubject;
+  final String? subject;
+  final List<Object>? topicLinks; // TODO shape of objects
+  final String? origContent;
+  final String? origRenderedContent;
+  final int? prevRenderedContentVersion;
```

Comment:
Hmm, the docs say "Clients should ignore this field."

I'd take that as a deprecation, even though it doesn't use the word "deprecated":
```suggestion
  // final int? prevRenderedContentVersion; // deprecated
```